docs/UserGuide.md

```diff
@@ -621,102 +513,118 @@ Important notes:
 > ```
-> Deleted Transaction:
-> 
-> Name: Lunch at McDonalds;
-> Type: expense;
-> Amount: $17.40;
-> Date: 15 Sep 2023 11:00;
-> Location: Clementi Mall;
-> Categories: #food
+> Invalid command format! 
+>
+> delete, delete_transaction, del: Deletes the transaction identified by the index number
+> used in the displayed transaction list.
+>
+> Argument: Index (must be a positive integer)
+>
+> Example: delete_transaction 1
 > ```
-> Input:
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteTransactionInitialState2.png" width="1000" />
-> Ouput:
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteTransactionSuccess2FinalState.png" width="1000" />
 
-##### Failed Execution
+**Example 2**
 
-###### Example 1
+> **Case**: Invalid `INDEX` provided
+> 
+> **Context**:`INDEX` given as `10` when only `5` transactions are present
+> in the current `Transaction List` configuration.
+>
+> **Input**: `delete_transaction 10`
+>
+> **Output**:
+> ```
+> The transaction index provided is invalid
+> ```
 
-> **Case**: Missing compulsory fields.
+**Example 3**
+
+> **Case**: Invalid `INDEX` provided
+> 
+> **Context**: `INDEX` given as a negative number
 >
-> **Input**: `delete_transaction`
+> **Inputs**: `delete_transaction -1`
 >
 > **Output**:
 > ```
 > Invalid command format! 
 >
-> delete_transaction: Deletes the transaction identified by the index number used in the displayed transaction list.
+> delete, delete_transaction, del: Deletes the transaction identified by the index number
+> used in the displayed transaction list.
 >
 > Argument: Index (must be a positive integer)
 >
 > Example: delete_transaction 1
 > ```
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteFail1.png" width="1000" />
 
-###### Example 2
+**Example 4**
 
-> **Case**: Invalid `<INDEX>` provided
-> (`<INDEX>` given as `10` given when only `5` transactions are presently displayed)
+> **Case**: Invalid `INDEX` provided
 >
-> **Input**: `delete_transaction 10`
+> **Context**: `INDEX` given as a number larger than `Integer.MAX_VALUE`
+>
+> **Inputs**: `delete_transaction 10000000000000000`
 >
 > **Output**:
 > ```
-> The transaction index provided is invalid
+> Invalid command format! 
+>
+> delete, delete_transaction, del: Deletes the transaction identified by the index number
+> used in the displayed transaction list.
+>
+> Argument: Index (must be a positive integer)
+>
+> Example: delete_transaction 1
 > ```
-> <img src="images/unicash/command-outputs/deleteTransaction/deleteFail2.png" width="1000" />
+
+
 
 #### Get Transaction
 
 Retrieves a `Transaction` from UniCa$h.
 
-Command: `get <INDEX>`
+Command: `get INDEX`
 
-Command Words Accepted: `get`, `g` (case-insensitive)
+Command Words Accepted: `get`, `g` (all case-insensitive)
 
-Command Argument: `<INDEX>` is the displayed transaction index
-of the transaction to be retrieved, as shown in the `Transactions List`.
+Command Argument: `INDEX` is the displayed transaction index
```

Comment:
Not to be "deleted" right? This is for get

Reply:
Oops my bad, fixed it